doc/user_guide/eds.rst

```diff
@@ -338,8 +392,21 @@ Selecting certain type of lines:
 Get lines intensity
 -------------------
 
-The :py:meth:`~._signals.eds.EDSSpectrum.get_lines_intensity` 
-method generates intensity maps by peak integration.
+
+The files needed for this section can be downloaded using
```

Comment:
I would also add a code line in the next example (I assume it's the core-shell sample) in line 415 (where some `spec_img` is used)